test/integration/targets/vmware_guest/tasks/mac_address_d1_c1_f0.yml

```diff
@@ -33,5 +33,5 @@
 - name: assert that changes were made
   assert:
     that:
-        - "clone_d1_c1_f0['instance']['hw_eth0']['addresstype'] == 'manual'"
-        - "clone_d1_c1_f0['instance']['hw_eth0']['macaddress'] == 'aa:bb:cc:dd:aa:42'"
+        - "clone_d1_c1_f0['instance']['Network Adapter 1']['addresstype'] == 'manual'"
```

Comment:
Tests would also need to be updated to match (4 lines altogether). This is likely the source of the CI failures on this PR.

Reply:
See comment above. Will try with underscores/lowercase. Thanks